Ex4.java

```diff
@@ -0,0 +1,47 @@
+public class Ex4 {
+
+    /**
+     * The fuction loop over the mapTiles array and count the number of islands
+     * @param numOfRow

```

Comment:
Please add comments to all parameters and return value.

Also minor stylistic comment: it's good to add empty line between main description of the method and the block of `@param` lines.